Frontend/main.py

```diff
@@ -19,33 +19,45 @@ def CenterWindowToDisplay(Screen: ctk, width: int, height: int, scale_factor: fl
 heading_label = ctk.CTkLabel(heading_frame, text="Encryption and Decryption", font=("Helvetica", 30))
 heading_label.pack(anchor=ctk.CENTER, fill=ctk.X)
 
-key_frame = ctk.CTkFrame(app)
-key_frame.pack(anchor=ctk.N, fill=ctk.X)
-
-def Ebrowse():
+def Ebrowsekey():
 
     filename = ctk.filedialog.askopenfilename(filetypes=[("Encryption", "*.txt*"), ("Decryption", "*.decrypted*")])
     encryptfilename.insert(0, filename)
-def Dbrowse():
+def Dbrowsekey():
```

Comment:
Please follow better code formatting and practices.
You can refer to https://peps.python.org/pep-0008/ for the same.

Reply:
okay